boards/peregrine/sam4l_wm400_cape/sam4l_wm400_cape.dts

```diff
@@ -0,0 +1,187 @@
+/*
+ * Copyright (c) 2020-2024 Gerson Fernando Budke <nandojve@gmail.com>
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <atmel/sam4lx4.dtsi>
+#include "sam4l_wm400_cape-pinctrl.dtsi"
+#include <zephyr/dt-bindings/input/input-event-codes.h>
+
+/ {
+	model = "Atmel SAM4L WM-400 Cape Board with an Atmel SAM4LC4B SoC";
+	compatible = "peregrine,sam4l_wm400_cape", "atmel,sam4lc4b", "atmel,sam4l";
+
+	aliases {
+		i2c-0 = &twim1;
+
+		led0 = &red_mod_led;
+		led1 = &green_mod_led;
+		led2 = &blue_mod_led;
+		led3 = &red_cape_led;
+
+		sw0 = &sw0_dfu;
```

Comment:
we don't do this for other boards with random newlines, why do we need this here?

Reply:
Because makes it more readable.